components/esp_wifi_remote/eppp/wifi_remote_rpc_server.cpp

```diff
@@ -0,0 +1,209 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <netdb.h>
+#include <memory>
+#include <cerrno>
+#include <sys/socket.h>
+#include "esp_log.h"
+#include "esp_check.h"
+#include "esp_tls.h"
+#include "esp_wifi.h"
+#include "wifi_remote_rpc_impl.hpp"
+#include "eppp_link.h"
+#include "wifi_remote_rpc_params.h"
+#include "lwip/apps/snmp.h"
+
+extern "C" esp_netif_t *wifi_remote_eppp_init(eppp_type_t role);
+
+namespace eppp_rpc {
+
+namespace server {
+const char *TAG = "rpc_server";
+
+const unsigned char ca_crt[] = "-----BEGIN CERTIFICATE-----\n" CONFIG_ESP_WIFI_REMOTE_EPPP_CLIENT_CA "\n-----END CERTIFICATE-----";
+const unsigned char crt[] = "-----BEGIN CERTIFICATE-----\n" CONFIG_ESP_WIFI_REMOTE_EPPP_SERVER_CRT "\n-----END CERTIFICATE-----";
+const unsigned char key[] = "-----BEGIN RSA PRIVATE KEY-----\n" CONFIG_ESP_WIFI_REMOTE_EPPP_SERVER_KEY "\n-----END RSA PRIVATE KEY-----";
+
+}
+
+using namespace server;
+
+class RpcInstance {
```

Comment:
Overall missing @briefs

Reply:
will add few doxygen comments, but this isn't a public API, so I'll keep it brief

Reply:
Added brief comments to the main classes and structs